src/scenarios/image/ImagePreview/ImagePreview.tsx

```diff
@@ -0,0 +1,224 @@
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { DocumentNode } from 'graphql';
+import { useState } from 'react';
+import {
+  Container,
+  Divider,
+  DropdownProps,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import 'shaka-player-react/dist/controls.css';
+import { getApolloClient } from '../../../apollo-client';
+import {
+  getEpisodeImagesQuery,
+  getMovieImagesQuery,
+  getSeasonImagesQuery,
+  getTvShowImagesQuery,
+} from './graphql-documents';
+
+interface Image {
+  id: string;
+  width: string;
+  height: string;
+  path: string;
+  type: string;
+}
+
+export const ImagePreview: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [entityId, setEntityId] = useState('');
+  const [images, setImages] = useState<Image[]>([]);
+  const [currentImageId, setCurrentImageId] = useState<string>();
+  const [width, setWidth] = useState<string>();
+  const [height, setHeight] = useState<string>();
+  const [path, setPath] = useState<string>();
+
+  const getImageUrl = async (): Promise<void> => {
+    const imageUrl = `${activeProfile.imageServiceBaseURL}${path}`;
+
+    logger.log('calling [getImageUrl]', 'output:', imageUrl);
+  };
+
+  const fetchImageForEntityId = async (): Promise<void> => {
```

Comment:
This can be renamed to `fetchImagesForEntityId` since an entity can have more than 1 images associated with it.

Reply:
makes sense to me :) updated now